Paco-Server/ear/default/web/js/controllers.js

```diff
@@ -722,63 +722,58 @@ pacoApp.controller('HackCtrl', ['$scope', '$http', function ($scope, $http) {
   $scope.hackRequest = '';
   $scope.errorResponse = '';
   $scope.submitForm = function () {
-    $http({
-      method: 'POST',
-      url: '/csSearch',
-      data: angular.fromJson($scope.hackRequest),
-      headers: {'Content-Type': 'application/json'}
+     $http({
+     method  : 'POST',
+     url     : '/csSearch?tz=America/Los_Angeles',
```

Comment:
Also, if we want to use the client side timezone, we either need to use offsets instead of ID for timezone, or, we might want to investigate the moments.js library which seems to offer IDs back. Not sure if that is worth exploring. Either, way it should probably be a separate feature branch.

Reply:
Actually, skip the moments.js we should be able to use the javascript api: Intl.DateTimeFormat().resolvedOptions().timeZone